### DISCUSSION TITLE
Erreur 502 endpoints API sandbox

### DISCUSSION THREAD

**Discussion by camillelenormand** [PayGreen/documentation#41] [Q&A] [answered]
Bonjour,

J'ai une erreur 502 depuis hier sur les endpoints (buyer, paymentorder) de votre API sandbox. Seul celui de l'authentification fonctionne.

Ci-dessous un exemple de réponse en échec:

```
<html><head>
<meta http-equiv="content-type" content="text/html;charset=utf-8">
<title>502 Server Error</title>
</head>
<body text=#000000 bgcolor=#ffffff>
<h1>Error: Server Error</h1>
<h2>The server encountered a temporary error and could not complete your request.<p>Please try again in 30 seconds.</h2>
<h2></h2>
</body></html>
```

Merci,

**Comment by AdrienBr** [comment, marked as answer]
Bonjour @camillelenormand ,
En effet notre environnement de Sandbox subit des perturbations depuis hier, nos équipes travaillent à résoudre le problème au plus vite.
 
Adrien
